framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/Deployment.java

```diff
@@ -39,6 +39,11 @@
      */
     void scale(int instances);
 
+    /**
+     * @return the current version of the deployment.
+     */
+    int getVersion();
```

Comment:
This method doesn't seem to be used anywhere.

Reply:
This was a leftover. I will remove it.